Build/Build.csproj

```diff
@@ -49,11 +49,11 @@
     <None Include="Remotion.Server.build" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="Remotion.BuildScript, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
why are the various dependencies missing here?

Reply:
I have no idea, that's not a change from this pull-request.

Reply:
This one's a remove-statement, so form the current PR. in the end, every package in packages.config must have a corresponding entry in build.csproj. This normally works without issue when updating the package via nuget command line.

Reply:
Okay, just noticed the update to the PR in the mail history :) please fix the remaining dependencies in the csproj file and both packages.config files. Given we're right now updating the build script, this change is directly related to the PR. I did not check which inconsistencies exist right now, but I think there was more than what got fixed in the udapte, correct?

Reply:
Actually, why does Build.proj need to have the references? I can only assume that Nuget/Visual Studio removed those references and didn't add the new ones because they are not needed by the code